### DISCUSSION TITLE
原来的vim默认<C-v>键，也就是ctrl+v按键是块模式中的列选择模式，现在变成了复制键，想恢复？

### DISCUSSION THREAD

**Discussion by wxzvim** [nshen/learn-neovim-lua#75] [Q&A]
原来的vim默认<C-v>键，也就是ctrl+v按键是块模式中的列选择模式，现在变成了复制键。
我想恢复默认设置，，我在nvim配置中没有找到相关的配置，怎么恢复来着？

**Comment by wxzvim** [comment]
https://blog.csdn.net/AngelLover2017/article/details/122070291?spm=1001.2101.3001.6661.1&utm_medium=distribute.pc_relevant_t0.none-task-blog-2%7Edefault%7EOPENSEARCH%7Edefault-1-122070291-blog-92015923.pc_relevant_multi_platform_whitelistv2eslanding&depth_1-utm_source=distribute.pc_relevant_t0.none-task-blog-2%7Edefault%7EOPENSEARCH%7Edefault-1-122070291-blog-92015923.pc_relevant_multi_platform_whitelistv2eslanding&utm_relevant_index=1

按照上面链接试了映射，根本不行

**Comment by nshen** [comment]
我这里ctrl v 是列选择。我觉得是你的终端带了快捷键吧？Terminal里边应该有快捷键设置，找到删掉试试。

**Reply by wxzvim** [reply]
的确如此，，改下json文件paste快捷方式为false就好了，感谢大佬！！！！
